.env.example

```diff
@@ -1,5 +1,27 @@
-ROSETTA_API_URL=
-DELIVERY_OPTIONS_API_URL=
-STAFFIN_IP_ADDRESSES=value1,value2
-ONSITE_IP_ADDRESSES=value1,value2
-DCS_PREFIXES=value1,value2
+# API Configuration
+ROSETTA_API_URL=https://api.example.com/rosetta
+WAGTAIL_API_URL=https://api.example.com/wagtail
+DELIVERY_OPTIONS_API_URL=https://api.example.com/delivery
+
+# API Timeout Settings (in seconds)
+# Set to empty or omit for no timeout (waits indefinitely)
+# WAGTAIL_API_TIMEOUT defaults to None if not set
+# ROSETTA_API_TIMEOUT defaults to None if not set
+# DELIVERY_OPTIONS_API_TIMEOUT defaults to 30 if not set
+# Wagtail API timeout (optional, defaults to None for no timeout)
+WAGTAIL_API_TIMEOUT=
+# Rosetta API timeout (optional, defaults to None for no timeout)
+ROSETTA_API_TIMEOUT=
+# Delivery Options API timeout (defaults to 30 seconds if not set)
+DELIVERY_OPTIONS_API_TIMEOUT=30
+
+# Maximum number of subjects listed in details page per record
+MAX_SUBJECTS_PER_RECORD=20
+
+# IP Address ranges (CIDR format) to determine if a user is staff or onsite (at Kew)
+STAFFIN_IP_ADDRESSES=1.2.3.4/22,5.6.7.8/13
+ONSITE_IP_ADDRESSES=10.11.12.13/20,20.21.22.23/24
+
+# Distressing content catalogue prefixes
+DCS_PREFIXES="ABC 1,DEF 21,GHI" # Must be comma separated if more than one.
+
```

Comment:
Only sensitive values need to go in the `.env` file.

These can all move to `docker-compose.yml`:

- `ROSETTA_API_URL`
- `WAGTAIL_API_URL`
- `DELIVERY_OPTIONS_API_URL`
- `WAGTAIL_API_TIMEOUT` (needs a default added to the config)
- `ROSETTA_API_TIMEOUT` (needs a default added to the config)
- `DELIVERY_OPTIONS_API_TIMEOUT` (needs a default added to the config)
- `MAX_SUBJECTS_PER_RECORD` (needs a default added to the config)
- `DCS_PREFIXES`